src/js/views/patients/schedule/schedule_views.js

```diff
@@ -0,0 +1,75 @@
+import { View } from 'marionette';
+import hbs from 'handlebars-inline-precompile';
+
+import intl from 'js/i18n';
+
+import 'sass/modules/list-pages.scss';
+import 'sass/modules/table-list.scss';
+
+import PreloadRegion from 'js/regions/preload_region';
+
+import Tooltip from 'js/components/tooltip';
+
+import LayoutTemplate from './layout.hbs';
+
+const LayoutView = View.extend({
+  className: 'flex-region',
+  template: LayoutTemplate,
+  regions: {
+    filters: '[data-filters-region]',
+    table: '[data-table-region]',
+    list: {
+      el: '[data-list-region]',
+      regionClass: PreloadRegion,
+    },
+    selectAll: '[data-select-all-region]',
+    title: '[data-title-region]',
+  },
+
+});
+
+const ScheduleTitleView = View.extend({
+  template: hbs`
+    {{formatMessage (intlGet "patients.schedule.scheduleViews.scheduleTitleView.title") role=role owner=owner}}{{~ remove_whitespace ~}}
+    <span class="list-page__header-icon js-title-info">{{fas "info-circle"}}</span>
+  `,
+  ui: {
+    tooltip: '.js-title-info',
+  },
+  templateContext() {
+    return {
+      owner: this.getOption('owner').get('name') || this.getOption('role').get('name'),
+    };
+  },
+  initialize({ state }) {
+    this.state = state;
+
+    this.listenTo(state, {
```

Comment:
as mentioned above.. we should either be doing this and passing the state down, or we should be re-showing it in the `restart`. I think this was copied from the worklist and should be cleaned up there as well